### PR TITLE
Update from EN docs for install-kubectl task

### PR DESCRIPTION
**Pull request by rekcah78**
Update from EN version

### CODE REVIEW

File: content/fr/docs/tasks/tools/install-kubectl.md [hunk 1]
```diff
@@ -80,28 +80,33 @@ yum install -y kubectl
 {{< /tab >}}
 {{< /tabs >}}
 
+### Installation avec des gestionnaires des paquets alternatif
```
Comment (rbenzair) [inline content/fr/docs/tasks/tools/install-kubectl.md]: alternatifs

File: content/fr/docs/tasks/tools/install-kubectl.md [hunk 2]
```diff
@@ -354,49 +374,69 @@ Les deux approches sont équivalentes. Après avoir rechargé votre shell, l'aut
...
 Comme indiqué dans la sortie de `brew install` (section "Caveats"), ajoutez les lignes suivantes à votre fichier `~/.bashrc` ou `~/.bash_profile` :
 
 ```shell
-[ -f /usr/local/etc/bash_completion ] && . /usr/local/etc/bash_completion
+export BASH_COMPLETION_COMPAT_DIR="/usr/local/etc/bash_completion.d"
+[[ -r "/usr/local/etc/profile.d/bash_completion.sh" ]] && . "/usr/local/etc/profile.d/bash_completion.sh"
 ```
 
-Rechargez votre shell.
+Rechargez votre shell et vérifié que bash-completion v2 est correctement installé avec `type _init_completion`.
```
Comment (sftim) [inline content/fr/docs/tasks/tools/install-kubectl.md]:
```suggestion
Rechargez votre shell et vérifiéz que bash-completion v2 est correctement installé avec `type _init_completion`.
```
?

File: content/fr/docs/tasks/tools/install-kubectl.md [hunk 3]
```diff
@@ -80,28 +80,33 @@ yum install -y kubectl
 {{< /tab >}}
 {{< /tabs >}}
 
+### Installation avec des gestionnaires des paquets alternatifs
```
Comment (remyleone) [inline content/fr/docs/tasks/tools/install-kubectl.md]:
```suggestion
### Installation avec des gestionnaires de paquets alternatifs
```

File: content/fr/docs/tasks/tools/install-kubectl.md [hunk 4]
```diff
@@ -354,49 +374,69 @@ Les deux approches sont équivalentes. Après avoir rechargé votre shell, l'aut
...
 Comme indiqué dans la sortie de `brew install` (section "Caveats"), ajoutez les lignes suivantes à votre fichier `~/.bashrc` ou `~/.bash_profile` :
 
 ```shell
-[ -f /usr/local/etc/bash_completion ] && . /usr/local/etc/bash_completion
+export BASH_COMPLETION_COMPAT_DIR="/usr/local/etc/bash_completion.d"
+[[ -r "/usr/local/etc/profile.d/bash_completion.sh" ]] && . "/usr/local/etc/profile.d/bash_completion.sh"
 ```
 
-Rechargez votre shell.
+Rechargez votre shell et vérifiéz que bash-completion v2 est correctement installé avec `type _init_completion`.
```
Comment (remyleone) [inline content/fr/docs/tasks/tools/install-kubectl.md]:
```suggestion
Rechargez votre shell et vérifiez que bash-completion v2 est correctement installé avec `type _init_completion`.
```

**Pull request merged**